packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganization.tsx

```diff
@@ -38,7 +38,7 @@ export const CreateOrganization = withOrganizationsEnabledGuard(
   { mode: 'redirect' },
 );
 
-export const CreateOrganizationModal = (props: CreateOrganizationProps): JSX.Element => {
+export const CreateOrganizationModal = (props: Omit<CreateOrganizationProps, 'path' | 'routing'>): JSX.Element => {
```

Comment:
❓ Should we create a `WithoutRouting<T> =  Omit<T, 'path' | 'routing'>` helper type to re-use it in all those cases?

Reply:
Added!